--- a/src/Objects/makefile_unix	2019-02-20 19:11:00.000000000 -0600
+++ b/src/Objects/makefile_unix	2019-02-20 19:13:43.000000000 -0600
@@ -10,8 +10,7 @@
 
 # Location of Python for 2.7 and 3.6 development headers for phcpy
 
-PYTHON=/usr/include/python2.7
-PYTHON3=/usr/local/include/python3.6m
+PYTHON3=$(SAGE_VENV)/include/python3.12
 
 # The location of the Sage Python lib:
 
@@ -24,7 +24,9 @@
 # GNAT Community 2019 :
 # ADALIB=/home/jan/opt/GNAT/2019/lib/gcc/x86_64-pc-linux-gnu/8.3.1/adalib
 # GNAT Community 2020 :
-ADALIB=/home/microway/opt/GNAT/2020/lib/gcc/x86_64-pc-linux-gnu/9.3.1/adalib
+# ADALIB=/home/microway/opt/GNAT/2018/lib/gcc/x86_64-pc-linux-gnu/7.3.1/adalib
+# For Ubuntu 22.04
+ADALIB=/usr/lib/gcc/x86_64-linux-gnu/10/adalib/
 
 # The location of the g++ compiler
 
@@ -41,8 +43,8 @@
 
 # Location of the QD library and its GPU version:
 
-QD_ROOT=/usr/local/qd-2.3.17
-QD_LIB=/usr/local/lib
+QD_ROOT=a
+QD_LIB=/home/sage/sage/local/lib
 GQD_HOME=/usr/local/gqd_1_2
 SDK_HOME=/usr/local/cuda/sdk
 
