--- a/src/snappea/dt2snap.cc
+++ b/src/snappea/dt2snap.cc
@@ -83,9 +83,9 @@
     int             theNeighbor[2][2], theNeighborStrand[2][2];
     Triangulation   *theTriangulation;
     
-    millettNeighbor = new ( int (*[crossing_number]) );
+    millettNeighbor = new int*[crossing_number];
     for ( i=0; i<crossing_number; ++i ) millettNeighbor[i] = new int[4];
-    millettNeighborView = new ( int (*[crossing_number]) );
+    millettNeighborView = new int*[crossing_number];
     for ( i=0; i<crossing_number; ++i ) millettNeighborView[i] = new int[4];
     
     crossing_sign = new int[crossing_number];
--- a/src/snappea/crossing_signs.cc
+++ b/src/snappea/crossing_signs.cc
@@ -20,14 +20,14 @@
         
     }
     
-    Pass **orig_pass = new ( Pass (*[ 2*crossing_number + 1 ]) );
-    Pass **new_pass_1 = new ( Pass (*[ num_components - 1 ]) );
-    Pass **new_pass_2 = new ( Pass (*[ num_components - 1 ]) );
+    Pass **orig_pass = new Pass*[ 2*crossing_number + 1 ];
+    Pass **new_pass_1 = new Pass*[ num_components - 1 ];
+    Pass **new_pass_2 = new Pass*[ num_components - 1 ];
     
-    Pass **join_1 = new ( Pass (*[ num_components - 1 ]) );
-    Pass **join_2 = new ( Pass (*[ num_components - 1 ]) );
-    Pass **join_3 = new ( Pass (*[ num_components - 1 ]) );
-    Pass **join_4 = new ( Pass (*[ num_components - 1 ]) );
+    Pass **join_1 = new  Pass*[ num_components - 1 ];
+    Pass **join_2 = new  Pass*[ num_components - 1 ];
+    Pass **join_3 = new  Pass*[ num_components - 1 ];
+    Pass **join_4 = new  Pass*[ num_components - 1 ];
     
     vector<int> a;
     vector<int> f;
